cmd/kube-apiserver/app/server.go

```diff
@@ -239,6 +239,7 @@ func (s *APIServer) AddFlags(fs *pflag.FlagSet) {
 	// Kubelet related flags:
 	fs.BoolVar(&s.KubeletConfig.EnableHttps, "kubelet-https", s.KubeletConfig.EnableHttps, "Use https for kubelet connections")
 	fs.UintVar(&s.KubeletConfig.Port, "kubelet-port", s.KubeletConfig.Port, "Kubelet port")
+	fs.MarkDeprecated("kubelet-port", "kubelet-port is deprecated and will be removed")
```

Comment:
Omit the date. We haven't decided on a policy yet. Additionally, we're going to deprecate all the flags hopefully in the 1.2 timeframe, so it doesn't seem worth it to make an extra effort for this one.